common/transport/mqtt/src/main/java/org/thingsboard/server/transport/mqtt/MqttTransportHandler.java

```diff
@@ -493,8 +499,13 @@ private void processDevicePublish(ChannelHandlerContext ctx, MqttPublishMessage
                 transportService.process(deviceSessionCtx.getSessionInfo(), rpcResponseMsg, getPubAckCallback(ctx, msgId, rpcResponseMsg));
             } else if (topicName.startsWith(MqttTopics.DEVICE_RPC_REQUESTS_TOPIC)) {
                 TransportProtos.ToServerRpcRequestMsg rpcRequestMsg = payloadAdaptor.convertToServerRpcRequest(deviceSessionCtx, mqttMsg, MqttTopics.DEVICE_RPC_REQUESTS_TOPIC);
-                transportService.process(deviceSessionCtx.getSessionInfo(), rpcRequestMsg, getPubAckCallback(ctx, msgId, rpcRequestMsg));
-                toServerRpcSubTopicType = TopicType.V1;
+                if (SERVICE_CONFIGURATION.equals(rpcRequestMsg.getMethodName())) {
+                    toServerRpcSubTopicType = TopicType.V1;
+                    onGetServiceConfigurationRpc(deviceSessionCtx.getSessionInfo(), ctx, msgId, rpcRequestMsg);
+                } else {
+                    transportService.process(deviceSessionCtx.getSessionInfo(), rpcRequestMsg, getPubAckCallback(ctx, msgId, rpcRequestMsg));
+                    toServerRpcSubTopicType = TopicType.V1;
```

Comment:
Can `toServerRpcSubTopicType = TopicType.V1;` be moved outside if-else? Or it is important for it to be assigned after `process` call?